docs/modules/nonlinearities.rst

```diff
@@ -3,5 +3,20 @@
 
 .. automodule:: lasagne.nonlinearities
 
+.. autosummary::
+
+   sigmoid
```

Comment:
I can imagine there are situations where we would want our docstring to differ from the one in Theano though, so maybe the wrapping approach is the safest bet :)

If you can't come up with a good docstring feel free to just put 'WRITEME' so we can handle it in a different PR.